bigtable-client-core-parent/bigtable-hbase/src/test/java/com/google/cloud/bigtable/hbase/wrappers/veneer/TestRowResultAdapter.java

```diff
@@ -119,38 +119,60 @@ public void testWhenRowKeyIsNotSet() {
 
   @Test
   public void testOnlyValueIsDifferent() {
```

Comment:
Can you add a separate test for dedupe functionality. It'll make it easier to debug if this ever gets messed up